### DISCUSSION TITLE
How to migrate Wallabag from Softaculous to standalone installation

### DISCUSSION THREAD

**Discussion by EdenSpire** [wallabag/wallabag#7390] [Q&A]
Hello,

Right now, I'm using Softaculous to install and maintain Wallabag, but the developer (Softaculous) is not very interested to maintain the Wallabag installation. I need to remind them most of the time whenever there is a new version.

My question is, can I migrate Wallabag from Softaculous? Can it be done?
Please help me.

Thanks in Advance!

**Comment by j0k3r** [comment]
Well I guess, you might be able to dump the database and use it in a new fresh installation.
But AFAIK nobdoy tried to move out from Softaculous

**Reply by EdenSpire** [reply]
Hello,

What's the easiest way to do that? Any guides for no techie user?

Thanks again.
